### ISSUE TITLE
Ci Input tool has narrow table

### ISSUE THREAD

**Issue by annamorphism** [DDMAL/CantusDB#639]
The table in the Input Tool pop-up is narrow, which looks silly and also means a lot of scrolling to find the relevant chant
<img width="1111" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/57a59806-b4ef-4edb-9672-c5a826f1799d">
cf Old Cantus: 
<img width="1184" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/d3cbc4d1-d111-4263-912e-69e8b43e026c">

**Comment by jacobdgm** [comment]
Which browser are you using, @annamorphism? I've tried this in Chrome, Firefox and Safari, on both Staging and Production, and all of them default to a fairly wide window, with a table that spans the full width of the window.

**Comment by annamorphism** [comment]
This is in Firefox! Hmm.

**Comment by jacobdgm** [comment]
I was working with @lucasmarchd01 on #637, and we noticed this. It was in Firefox, so maybe it's a firefox thing? Let me try again to recreate it.

Update: yes, I'm able to recreate this in Firefox.
